### PR TITLE
get fill transparency of edit geom from preference store

### PR DESCRIPTION
**Pull request by sloob**
Adds the possibility to specify the factor to reduce the transparency of the color used to fill the geoms on the EditBlackboard.

Signed-off-by: sloob <sebastian.loob@ibykus.de>

### CODE REVIEW

File: plugins/org.locationtech.udig.tool.edit/src/org/locationtech/udig/tool/edit/internal/messages.properties [hunk 1]
```diff
@@ -116,6 +116,8 @@ EditToolPreferences_snapRadius = Snap Radius
 
 EditToolPreferences_vertexDiameter = Vertex Diameter
 
+EditToolPrefernece_fillTransparency = Reduce Fill Transparency (0-100)
```
[the remark this answers is not in the document]
Comment (fgdrf) [reply plugins/org.locationtech.udig.tool.edit/src/org/locationtech/udig/tool/edit/internal/messages.properties]: Good point, just had a look at Style Dialogs and I agree, we should have a consitent usage within uDig:

grafik

Let's use "Fill Opacity" for now and "Vertex Opacity" for #324

**Comment by fgdrf** [comment]
While submitting the other pull request I noticed a missmatch betwenn labeled "Opacity" and internal usage of variables "transparency". 

because #313 is submitted we have merge conflictes for this change. Can you please resolve these and rename variables and properties?

Many Thanks!

File: plugins/org.locationtech.udig.tool.edit/src/org/locationtech/udig/tool/edit/internal/messages.properties [hunk 2]
```diff
@@ -116,7 +116,7 @@ EditToolPreferences_snapRadius = Snap Radius
 
 EditToolPreferences_vertexDiameter = Vertex Diameter
 
-EditToolPrefernece_fillTransparency = Reduce Fill Transparency (0-100)
+EditToolPrefernece_fillTransparency = Fill Opacity
```
Comment (fgdrf) [inline plugins/org.locationtech.udig.tool.edit/src/org/locationtech/udig/tool/edit/internal/messages.properties]: Would you refactor this property to `EditToolPreferences_fillOpacity` and update screenshot for EditTool Preferences page (doc).

Comment (sloob) [reply plugins/org.locationtech.udig.tool.edit/src/org/locationtech/udig/tool/edit/internal/messages.properties]: hi @fgdrf,
yes i can do that. I try to fix it this week.

**Comment by fgdrf** [comment]
@sloob Again, many thanks for this improvement!

File: plugins/org.locationtech.udig.tool.edit/src/org/locationtech/udig/tool/edit/internal/messages.properties (hunk 2 above)
Comment (sloob) [reply plugins/org.locationtech.udig.tool.edit/src/org/locationtech/udig/tool/edit/internal/messages.properties]: the last commit also fixes the misspelling variable for the vertex opacity, so i think #328 is not needed anymore

**Pull request merged**